docs/api.md

```diff
@@ -7,26 +7,144 @@ Loosely based off the example in the README
 
 | Name | Type | Description | Example |
 | ------- | ------- | ------- | ------- |
-| **[black_id](#resource-player)** | *number* | this could be a uuid to make it db agnostic | `42.0` |
-| **date_played** | *date-time* |  | `"2015-01-01T12:00:00Z"` |
-| **game_record** | *string* |  | `"<raw sgf string>"` |
-| **game_server** | *string* |  | `"KGS"` |
-| **rated** | *boolean* |  | `true` |
-| **[white_id](#resource-player)** | *number* | this could be a uuid to make it db agnostic | `42.0` |
+| **[black_id](#resource-player)** | *integer* | black player's id | `42` |
+| **date_played** | *date-time* | time the game was played | `"2015-01-01T12:00:00Z"` |
+| **game_record** | *string* | game record in SGF | `"<raw sgf string>"` |
+| **game_server** | *string* | game server id | `"KGS"` |
+| **id** | *integer* | this could be a uuid to make it db agnostic | `42` |
+| **rated** | *boolean* | whether this was a rated game (on the server?) | `true` |
+| **[white_id](#resource-player)** | *integer* | white player's id | `42` |
+
+### Game Creation
+
+Report a game result
+
+```
+POST /api/v1/games
+```
+
+#### Required Parameters
+
+| Name | Type | Description | Example |
+| ------- | ------- | ------- | ------- |
+| **black_id** | *integer* | black player's id | `42` |
+| **date_played** | *date-time* | time the game was played | `"2015-01-01T12:00:00Z"` |
+| **game_record** | *string* | game record in SGF | `"<raw sgf string>"` |
+| **game_server** | *string* | game server id | `"KGS"` |
+| **rated** | *boolean* | whether this was a rated game (on the server?) | `true` |
+| **white_id** | *integer* | white player's id | `42` |
+
+
+
+#### Curl Example
+
+```bash
+$ curl -n -X POST http://dev.usgo.org/api/v1/games \
+  -d '{
+  "black_id": 42,
+  "white_id": 42,
+  "game_server": "KGS",
+  "rated": true,
+  "date_played": "2015-01-01T12:00:00Z",
+  "game_record": "<raw sgf string>"
+}' \
+  -H "Content-Type: application/json" \
+  -H "X-Auth-Server-Token: <secret server token>" \
+  -H "X-Auth-Black-Player-Token: <secret token for black player>" \
+  -H "X-Auth-White-Player-Token: <secret token for white player>"
+```
+
+
+#### Response Example
+
+```
+HTTP/1.1 201 Created
+```
+
+```json
+{
+  "id": 42
+}
+```
+
+### Game Read
```

Comment:
Ah, I think I didn't explain properly. Doing 
`GET /api/v1/games/17` will return a json blob describing the game object. Doing `GET /api/v1/games/17/sgf` will return the raw SGF file. In other words, putting that URL into your browser would cause your browser to download the file, and you could open that file in your favorite SGF editor.